content/blog/2020/12/2020-12-04-3-Cases:-Jenkins-Success-stories-from-the-community.adoc

```diff
@@ -0,0 +1,56 @@
+
+Of course, they had to build and release their software framework (gCube) in a way that would support multi-project releases at scale — from 200+ Git repositories within the same day! It had to be fast, automate all release activities, and it had to deliver incremental releases to address user requirements quickly. Most of all, the solution had to be cost-effective.
+
+Using Jenkins, they created an innovative approach to software delivery: a continuous integration/continuous delivery (CI/CD) pipeline, scalable, easy to maintain, and upgradable at a minimal cost.  
+
+Discover how D4Science empowers e-Science and virtual research communities with software released via Jenkins. link:https://jenkinsistheway.io/case-studies/d4science-amps-up-their-scientific-research-platform-with-ci-cd-powered-by-jenkins/[Read the Jenkins case study featuring D4Science here!]
+
+== Case Study #2: Gainsight
+
+=== Humanizing CSX with tech innovation and a robust DevSecOps platform
+
+link:https://www.gainsight.com/[Gainsight's] customer service experience platform helps customer success teams at more than 100 leading IT and healthcare clients. How? By driving engagement for tens of thousands of their customers. 
+
+That's why the engineering team at Gainsight approached the customer experience by building a smarter, faster DevSecOps platform using Jenkins. They stuck to an infrastructure-as-code approach while integrating various tools and programming languages all within the platform. And they secured processes with better visibility and air-tight quality control.
+
+The result was a flexible DevSecOps infrastructure, 95% of which is scalable with code. And the cost of infrastructure costs was 40% less. That provides Gainsight with ease of collaboration, keener operational insight, and — because builds are 30% faster — the ability to stay a step ahead of the competition.
+
+Read why Gainsight's lead DevOps engineer, Prudviraj Pentakota, says "Jenkins is the epicenter of DevSecOps in our organization." link:https://jenkinsistheway.io/case-studies/jenkins-case-study-gainsight/[Get the full story here.]
+
+== Case Study #3: Avoris Travel
+
+=== Reinventing travel with an inventive technology platform
+
+Part of Barceló Group, link:https://www.avoristravel.com/[Ávoris Travel] is behind prominent destination travel brands like LeSki, Le Musik, and a selection of author travels under its "Viagens Com Assinatura" signature travel concept. A proprietary database and a smart, dynamic booking engine are the tickets to offering differentiating and inventive travel opportunities.
+
+Also unique to Avoris is a discreet machining technology that enables agents to enter specific criteria to search and find all types of trips and travel opportunities across the entire network. 
```

Comment:
"Machining" is usually associated with cutting metal, while "matching" is associated with search criteria.  The text "machiniing" matches the [upstream site](https://jenkinsistheway.io/case-studies/jenkins-case-study-avoris-travel/), but is surprising there as well.
```suggestion
Also unique to Avoris is a discreet matching technology that enables agents to enter specific criteria to search and find all types of trips and travel opportunities across the entire network. 
```